azext_edge/edge/providers/check/base/deployment.py

```diff
@@ -134,3 +164,51 @@ def _check_k8s_version(as_list: bool = False) -> Dict[str, Any]:
         )
 
     return check_manager.as_dict(as_list)
+
+
+def _check_storage_classes(acs_config: dict, as_list: bool = False) -> Dict[str, Any]:
+    from kubernetes.client.models import V1StorageClassList
+
+    expected_classes = acs_config.get("feature.diskStorageClass", [])
```

Comment:
does default empty list make sense when the expected value is a comma delimited value str?